feature-libs/checkout/schematics/add-checkout/index.ts

```diff
@@ -39,43 +163,42 @@ export function addCheckoutFeatures(options: SpartacusCheckoutOptions): Rule {
     return chain([
       addPackageJsonDependenciesForLibrary(peerDependencies, options),
 
-      shouldAddFeature(CLI_CHECKOUT_FEATURE, options.features)
-        ? addSavedCheckoutFeature(options)
-        : noop(),
+      determineCheckoutFeatures(options, packageJson),
     ]);
   };
 }
 
-function addSavedCheckoutFeature(options: SpartacusCheckoutOptions): Rule {
-  return addLibraryFeature(options, {
-    folderName: CHECKOUT_FOLDER_NAME,
-    moduleName: CHECKOUT_MODULE_NAME,
-    featureModule: {
-      name: CHECKOUT_MODULE,
-      importPath: SPARTACUS_CHECKOUT,
-    },
-    rootModule: {
-      name: CHECKOUT_ROOT_MODULE,
-      importPath: SPARTACUS_CHECKOUT_ROOT,
-    },
-    lazyLoadingChunk: {
-      moduleSpecifier: SPARTACUS_CHECKOUT_ROOT,
-      namedImports: [CHECKOUT_FEATURE_NAME_CONSTANT],
-    },
-    i18n: {
-      resources: CHECKOUT_TRANSLATIONS,
-      chunks: CHECKOUT_TRANSLATION_CHUNKS_CONFIG,
-      importPath: SPARTACUS_CHECKOUT_ASSETS,
-    },
-    styles: {
-      scssFileName: SCSS_FILE_NAME,
-      importStyle: SPARTACUS_CHECKOUT,
-    },
-    dependencyManagement: {
-      featureName: CLI_CHECKOUT_FEATURE,
-      featureDependencies: {
-        [SPARTACUS_USER]: [CLI_USER_ACCOUNT_FEATURE],
-      },
-    },
-  });
+function determineCheckoutFeatures(
+  options: SpartacusCheckoutOptions,
+  packageJson: any
+): Rule {
+  if (
+    shouldAddFeature(
+      CLI_CHECKOUT_SCHEDULED_REPLENISHMENT_FEATURE,
+      options.features
+    )
+  ) {
+    return chain([
+      addLibraryFeature(options, checkoutScheduleReplenishmentFeatureConfig),
+      addFeatureTranslations(options, checkoutBaseFeatureConfig),
+      addFeatureTranslations(options, checkoutB2bFeatureConfig),
+
+      configureB2bFeatures(options, packageJson),
```

Comment:
😉 

gotta remove that `defaultB2bCheckoutConfig` from getB2bConfiguration

Reply:
Done 😎